packages/data-stores/src/plans/test/resolvers.ts

```diff
@@ -30,6 +13,8 @@ jest.mock( 'wpcom-proxy-request', () => ( {
 	__esModule: true,
 } ) );
 
+// const TEST_LOCALE = 'test-locale';
+
 describe( 'getSupportedPlans', () => {
 	it( 'calls setFeatures, setFeaturesByType, and setPlans after fetching plans', () => {
```

Comment:
According to strict Unit Testing rules, these should be 3 separate tests. What do you think?

Reply:
I see your point, but I'm not sure how to split this test, since it relies on testing the resolver as a generator function, calling `next() on each.

For now, I've wrapped this test into one more resolvers `describe()` call, and renames the test to a better name in https://github.com/Automattic/wp-calypso/pull/50492/commits/6a2c1cb1af6a86e1cd6f11c8fe520b415f02aef2